geojson/base.py

```diff
@@ -144,7 +139,3 @@ def errors(self):
         # make sure that each subclass implements it's own validation function
         if self.__class__ != GeoJSON:
             raise NotImplementedError(self.__class__)
-
-        # check for errors on own dict (self)
-        results = {key: obj.errors() for (key, obj) in self.iteritems()}
-        return {key: err for (key, err) in results.iteritems() if err}
```

Comment:
And this isn't actually unreachable (you can get to it if you call `errors()` on a `GeoJSON` object -- but it's broken, and throws an exception because `self.iteritems()` returns values which aren't themselves `GeoJSON` subclasses, so they don't have `errors()` methods themselves.

So it seemed better to just have `GeoJSON().errors()` always return `None`, as this does, implicitly.